backend/routes/admins.js

```diff
@@ -1,27 +1,33 @@
 const express = require('express');
 const router = express.Router();
-const db = require('../config/database');
-const User = require('../models/User');
-const jwt = require('jsonwebtoken');
 const authUser = require('../middleware/authUser');
 const adminController = require('../controllers/adminController');
-const authController = require('../controllers/authController');
-const authJwt = require('../middleware/authJwt');
-const constants = require("../utils/constants");
+const constants = require('../utils/constants');
 
 router.use(express.json());
 router.use(authUser.setUser);
 
 //Admin test page
-router.get('/', authUser.verifyUser, authUser.verifyRole(constants.ROLE.ADMIN), (req, res) => {
-    res.send("If you can access this then you are an admin !");
-});
+router.get(
+  '/',
+  authUser.verifyUser,
+  authUser.verifyRole(constants.ROLE.ADMIN),
+  (req, res) => {
+    res.send('If you can access this then you are an admin !');
+  }
+);
 
 //Update user roles as admin
-router.put('/updateRole', authUser.verifyUser, authUser.verifyRole(constants.ROLE.ADMIN), (req, res) => {
-    adminController.updateRole(req,res);
-})
+//TODO Fix route, 403 status code
```

Comment:
I found the 403 status code, it's associated with the middleware, due to the naming convention difference. Fixing the req,body naming convention from req.body.AccountId to req.body accountId should fix this.

Reply:
Turns into 401 with the naming convention changes

Reply:
It's the error of one of the comments I mentionned above in the file adminController.js: Tested the update function locally, currently not working properly due to line 16. The accountId of the user should be used to verify if the user has the rights to update roles as an admin. The email should be used in this case to properly update the role of a user.